README.md

```diff
@@ -21,7 +21,7 @@ Include a minified browser bundle directly in your HTML like so:
 ```html
 <script
   src="https://unpkg.com/algosdk@v2.3.0/dist/browser/algosdk.min.js"
```

Comment:
The version needs to be updated here as well (also applies a few lines down)
```suggestion
  src="https://unpkg.com/algosdk@v2.4.0/dist/browser/algosdk.min.js"
```

Reply:
Thank you